tunnel/intercept/interceptor.go

```diff
@@ -136,14 +136,15 @@ func (m LUT) GetByAddress(addr net.Addr) (*InterceptedService, error) {
 	return &s, nil
 }
 
-func (m LUT) GetByName(serviceName string) (*InterceptedService, error) {
+func (m LUT) GetByName(serviceName string) []InterceptedService {
+	r := make([]InterceptedService, 0)
```

Comment:
Best practice is to start with nil slice, since len and append both work with nil slices. 

`var r []InterceptedService`

At least that's what the linter always tells me :)

Reply:
Done, thanks! golang is similar enough to C that I can sometimes forget where I am :)